pymove/visualization/matplotlib.py

```diff
@@ -0,0 +1,199 @@
+import matplotlib.pyplot as plt
+
+from pymove.utils import constants
+from pymove.utils.constants import (
+    COUNT,
+    DATE,
+    DATETIME,
+    DAY,
+    HOUR,
+    LATITUDE,
+    LONGITUDE,
+    PERIOD,
+    SITUATION,
+    STOP,
+    TILES,
+    TRAJ_ID,
+)
+
+
+def show_object_id_by_date(
+    move_data,
+    create_features=True,
+    kind=None,
+    figsize=(21, 9),
+    return_fig=True,
+    save_fig=True,
+    name='shot_points_by_date.png',
+):
+    """Generates four visualizations based on datetime feature:
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5714604)